packages/api/src/platforms/vtex/index.ts

```diff
@@ -35,12 +35,13 @@ export interface Options {
   channel: string
   locale: string
   hideUnavailableItems: boolean
-  incrementAddress: boolean,
+  incrementAddress: boolean
   flags?: FeatureFlags
 }
 
 interface FeatureFlags {
   enableOrderFormSync?: boolean
+  optimisticCart?: boolean
```

Comment:
I think this flag should not live here, as it doesn't affect the behavior of the API. This also means undoing the changes you've done to API files because of this change.